pkg/reconciler/v1alpha1/revision/resources/constants.go

```diff
@@ -37,7 +37,7 @@ const (
 	autoscalerPort = 8080
 
 	// ServicePortName is the name of the external port of the service
-	ServicePortName = "http"
+	ServicePortName = "http2"
```

Comment:
Do these port name changes impact upgradability? Will those be reconciled correctly after upgrading Knative?